fix(tab): container tab usage (#601)
carbon-design-system/carbon-website: src/pages/components/tabs/usage.mdx

> _Tabs_ are used to quickly navigate between views within the same context.

Reviewer: ```suggestion
Tabs are used to quickly navigate between views within the same context.
```

<Row>
<Column colLg={8}>

![An example of tabs being used.](images/tab-usage-1.png)

</Column>
</Row>

## Format

> #### Label

Reviewer: ```suggestion
#### Tab label
```

> Each tab label describes its content and sets user expectations. Labels are concise and use one to two words maximum. Keep in mind that at mobile widths, the character length of a label will impact the experience. Icons are not permitted in tab labels.

Reviewer: ```suggestion
Each tab label describes the content contained within it. Labels are concise and use no more than two words. Keep in mind that at mobile widths and during translation, the character length of a label will impact the experience. Do not use icons in tab labels.
```

#### Number of tabs

> A maximum of six tabs is recommended for most scenarios. This is to maintain an uncluttered UI and reduce cognitive load for users. If more than six tabs are needed then consider other navigation patterns, such as a [side-nav](/components/UI-shell-left-panel/code).

Reviewer: ```suggestion
In most scenarios, you should use no more than six tabs. This maintains an uncluttered UI and reduces cognitive load for users. If more than six tabs are needed, consider other navigation patterns, such as a [side-nav](/components/UI-shell-left-panel/code).
```
